kahuna/public/js/preview/image.html

```diff
@@ -46,6 +46,13 @@
 
     <div class="preview__info" ng:if="! ctrl.hideInfo">
         <div class="flex-container">
+            <span class="preview__info__collection"
+                ng-repeat="collection in ctrl.image.data.collections"
+                gr:tooltip="{{collection.data.path.join(' ▸ ')}}">
```

Comment:
The [titip library](http://quantumui.org/appdoc/documents/home/titip) suggests doing:

```html
<div class="titip-right">
  foo bar
  <span class="titip-content">
    <h2> tooltip </h2>
    this is the tooltip!
  </span>
</div>
```

i.e. what ever has the class `titip-content` is the tooltip content.